programs/sbf/rust/big_mod_exp/Cargo.toml

```diff
@@ -1,19 +1,19 @@
 [package]
 name = "solana-bpf-rust-big-mod-exp"
-version = "1.16.0"
-description = "Solana BPF test program written in Rust"
-authors = ["Solana Labs Maintainers <maintainers@solanalabs.com>"]
-repository = "https://github.com/solana-labs/solana"
-license = "Apache-2.0"
-homepage = "https://solanalabs.com/"
 documentation = "https://docs.rs/solana-bpf-rust-big-mod-exp"
-edition = "2021"
+version = { workspace = true }
+description = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
 
 [dependencies]
-array-bytes = "=1.4.1"
-serde = { version = "1.0.112", features = ["derive"] }
-serde_json = "1.0.56"
```

Comment:
both of these will be updated by moving to the workspace version. probably not a problem, just wanted to raise